.github/workflows/cicd.yml

```diff
@@ -178,17 +178,27 @@ jobs:
 
     needs: [lint, mypy, docs, packaging]
```

Comment:
This is a question of taste.

I think that is good to reduce wasted test runs... but quite often I want to see the full errors of a commit, so that I can fix them all at once and then push a new general fix.

But for example, I have workflows in which if the Ubuntu tests fail, Win and macOS tests are not executed (to save money and save the environment)

I would prefer to run the ubuntu tests, even if mypy is not 100% happy :)

Reply:
In Travis, I had it set up so 1-2 unit test runs would go anyway, but that wasn't easily translated to GHA.

I may figure that out again later, but at the moment, this seems good enough.

I don't know if it's an issue with GHA, but I always found it annoying that I couldn't get a Travis result for a push because a million Twisted builds were queued up and many of them were wasting cycles on already-failed builds, and I think waste is waste so if you can't even mass listing steps, no need to try the unit test runs; it's not meaningfully slowing me down here.

Reply:
Also: all of those steps run pretty quickly locally.